src/Ryujinx.Ui.Common/Helper/ShortcutHelper.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Ryujinx.Common.Configuration;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+using System;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Runtime.Versioning;
+using System.Text;
+using Image = System.Drawing.Image;
+
+namespace Ryujinx.Ui.Common.Helper
+{
+    public static class ShortcutHelper
+    {
+        [SupportedOSPlatform("windows")]
+        private static void CreateShortcutWindows(string appFilePath, byte[] iconData, string iconPath, string cleanedAppName, string basePath, string desktopPath)
+        {
+            MemoryStream iconDataStream = new(iconData);
+            using (Image image = Image.FromStream(iconDataStream))
+            {
+                using Bitmap bitmap = new(128, 128);
+                using System.Drawing.Graphics graphic = System.Drawing.Graphics.FromImage(bitmap);
+                graphic.InterpolationMode = InterpolationMode.HighQualityBicubic;
+                graphic.DrawImage(image, 0, 0, 128, 128);
+                SaveBitmapAsIcon(bitmap, iconPath + ".ico");
+            }
+
+            IShellLink shortcut = (IShellLink)new ShellLink();
+
+            shortcut.SetDescription(cleanedAppName);
+            shortcut.SetPath(basePath + ".exe");
+            shortcut.SetIconLocation(iconPath + ".ico", 0);
+            shortcut.SetArguments($"""{basePath} "{appFilePath}" --fullscreen""");
```

Comment:
```suggestion
            shortcut.SetArguments($"""{basePath} --fullscreen "{appFilePath}"""");
```
Aside from the order of the arguments, could you use a list here instead?